yt_idefix/data_structures.py

```diff
@@ -724,3 +816,290 @@ def _validate_units_override_keys(cls, units_override):
             )
 
         super(cls, cls)._validate_units_override_keys(units_override)
+
+
+class PlutoXdmfDataset(PlutoVtkDataset):
+    _index_class = PlutoXdmfHierarchy
+    _field_info_class = PlutoXdmfFields
+    _dataset_type = "pluto-xdmf"
+    _version_regexp = re.compile(r"\d+\.\d+\.?\d*[-\w+]*")
```

Comment:
This appears to be identical to the class attributed inherited from `PlutoVtkDataset`
```suggestion
```